configs/body_2d_keypoint/topdown_heatmap/mpii/td-hm_mobilenetv2_8xb64-210e_mpii-256x256_badcase.py

```diff
@@ -0,0 +1,124 @@
+_base_ = ['../../../_base_/default_runtime.py']
+
+# runtime
+train_cfg = dict(max_epochs=210, val_interval=10)
+
+# optimizer
+optim_wrapper = dict(optimizer=dict(
+    type='Adam',
+    lr=5e-4,
+))
+
+# learning policy
+param_scheduler = [
+    dict(
+        type='LinearLR', begin=0, end=500, start_factor=0.001,
+        by_epoch=False),  # warm-up
+    dict(
+        type='MultiStepLR',
+        begin=0,
+        end=210,
+        milestones=[170, 200],
+        gamma=0.1,
+        by_epoch=True)
+]
+
+# automatically scaling LR based on the actual training batch size
+auto_scale_lr = dict(base_batch_size=512)
+
+# hooks
+default_hooks = dict(checkpoint=dict(save_best='PCK', rule='greater'),
+                     badcase=dict(type="BadCaseAnalyzeHook", 
+                                #   metric_type="loss",
+                                  metric_type="accuracy",
+                                  show=True,
+                                  badcase_thr=100,
+                                  out_dir='badcase'))
```

Comment:
How about moving `badcase` into `default_runtime.py` and disable by defaults? In this way, there is no need to add such a new config

Reply:
Is the config of `badcase` in experiment config files able to overwrite that in default_runtime.py?